cruise.umple/test/cruise/umple/implementation/requirements/SortReqsImplsOutside.ump

```diff
@@ -0,0 +1,17 @@
+// To test if the requirements are sorted correctly when the implementsReq is outside the class.
+req B01 {      
+  a is a second requirement statement.
+}    
+req A02 {    
+  a requirement.
+}
+req A01 {    
+  z requirement.
+}
+  implementsReq A02;
```

Comment:
This should be triggering the requirements doc to state that the requirement is implemented by class Example.